client_test.go

```diff
@@ -513,3 +435,290 @@ func TestGetFeature(t *testing.T) {
 	assert.Equal(t, ProjectUUID, feature.ProjectUUID)
 
 }
+
+const GetMVFeatureOptionResponseJson = `
+{
+    "id": 136,
+    "uuid": "8d3512d3-721a-4cae-9855-56c02cb0afe9",
+    "type": "unicode",
+    "string_value": "option_value_30",
+    "boolean_value": null,
+    "default_percentage_allocation": 60.0,
+    "feature": 1
+}
+`
+const MVFeatureOptionUUID = "8d3512d3-721a-4cae-9855-56c02cb0afe9"
+const MVFeatureOptionID int64 = 136
```

Comment:
Is this just an arbitrary number? If so, it seems a little odd to choose 136 - makes it seem like it's actually using some external source for data.

Reply:
Yep, good catch

Reply:
Can we not use `1` ? `150` still seems a little specific to me?

Reply:
1 is already being used by `feature ID` I'd like to avoid using the same one, give me a bit more confidence

Reply:
Yep, fair point. Why not `2` then? 😄

Reply:
Well, I am saving that for feature ID 2

Reply:
Ok, then leave it at 150 but maybe add a quick comment?

Reply:
What if I make it 200?